packages/-ember-decorators/tests/unit/object/computed-test.js

```diff
@@ -425,7 +425,7 @@ module('@computed', function() {
   module('modifiers', function() {
     test('volatile', function(assert) {
       assert.expect(2);
-      class Foo {
```

Comment:
Why do we need observers in this test? It isn't obvious to me...

Reply:
Ensuring that `volatile` works correctly. One of the things it does is make a property unobservable, so observers won't trigger.